Tests.Unit.Core.ApplicationServices/DomainServices/Role/RoleAssignmentsServiceTest.cs

```diff
@@ -1,31 +1,35 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
 using Core.DomainModel;
 using Core.DomainModel.GDPR;
 using Core.DomainModel.Result;
 using Core.DomainServices;
-using Core.DomainServices.GDPR;
 using Core.DomainServices.Options;
+using Core.DomainServices.Role;
 using Infrastructure.Services.Types;
 using Moq;
 using Tests.Toolkit.Patterns;
 using Xunit;
 
-namespace Tests.Unit.Core.DomainServices.GDPR
+namespace Tests.Unit.Core.DomainServices.Role
 {
-    public class DataProcessingRegistrationRoleAssignmentsServiceTest : WithAutoFixture
+    public class RoleAssignmentsServiceTest : WithAutoFixture
```

Comment:
Det er jo stadig en konket inkarnation af den så længe den peger på DPR objekter. Så ret beset bør vi nok lave den abstrakt (og generic) og så lave to nedarvninger - 1 til itsystemusage og 1 til dataprocessingregistration